varats/paper/paper_config.py

```diff
@@ -86,18 +109,29 @@ def multi_case_study_rev_filter(revision: str) -> bool:
     def add_case_study(self, case_study: CaseStudy) -> None:
         """
         Add a new case study to this paper config.
+
+        Args: The case study to add.
         """
         self.__case_studies[case_study.project_name] += [case_study]
 
+    def add_artefact(self, artefact: Artefact) -> None:
+        """
+        Add a new artefact to this paper config.
+
+        Args:
+            artefact: The artefact to add.
```

Comment:
```suggestion
            artefact: the artefact to add
```
???
I don't write full sentences for the param desc, I normally even try to reuse the param name in a quick desc
Do you think it is necessary to write full sentences here?

Reply:
Mostly personal preference I guess.
However, if you look at this [example](https://vara.readthedocs.io/en/vara-dev/vara-ts-api/paper.html#varats.paper.case_study.generate_case_study), I find it better readable if the description does not reuse the parameter name.